rna_transcription_updated.rb

```diff
@@ -0,0 +1,30 @@
+class Complement
+  def self.of_dna(dna)
+    dna=dna.split('');
+    rna=''
+    (0..dna.length-1).each do |i|
+      case dna[i]
+        when 'A'
+          rna = rna + 'U'
+          next
+        when 'G'
+          rna = rna + 'C'
+          next
+        when 'C'
+          rna = rna + 'G'
+          next
+        when 'T'
+          rna = rna + 'A'
+          next
+        else
+          rna = ''
+          break
+      end
```

Comment:
sir i want if when condition is true compiler should not check other condition it should continue with other iteration so i put next